src/Lumi/Components/Table/FilterDropdown.purs

```diff
@@ -106,19 +108,23 @@ filterItem_ = makeStateless filterItemComponent render
                   connectDragSource $ connectDropTarget $
                     element (unsafeCreateDOMComponent "lumi-row")
                       { className: if item.hidden then "" else "active"
-                      , style: css
-                          { padding: "0 8px 0 0"
-                          , alignItems: "center"
-                          , borderTop:
-                              if isOver && (fromMaybe false ((\dragItem -> dragItem.index > index) <$> maybeDragItem))
-                              then "2px solid #0044e4"
-                              else "2px solid transparent"
-                          , borderBottom:
-                              if isOver && (fromMaybe false ((\dragItem -> dragItem.index < index) <$> maybeDragItem))
-                              then "2px solid #0044e4"
-                              else "2px solid transparent"
-                          , opacity: if isDragging then 0.1 else 1.0
-                          }
+                      , style:
+                        let
+                          borderStyle :: (Int -> Int -> Boolean) -> String
+                          borderStyle compare' =
+                            if isOver && (fromMaybe false ((\dragItem -> dragItem.index `compare'` index) <$> maybeDragItem))
+                            then "2px solid " <> cssStringHSLA colors.primary
+                            else "2px solid " <> cssStringHSLA colors.transparent
+                          borderTop = borderStyle (>)
+                          borderBottom = borderStyle (<)
+                        in
+                          css
+                            { padding: "0 8px"
```

Comment:
This padding is the primary change -- the checkbox input used to have built in padding. The rest is just removing some duplication and using the colors module.

Reply:
Did we change the padding from `padding: "0 8px 0 0"` to `padding: "0 8px"` for a particular reason?

Reply:
Yes, the checkbox input used to have built in padding (margin), so this parent only specified padding on the right edge. Now the checkbox has no build-in padding, so this parent specifies both sides.